README.md

```diff
@@ -6,7 +6,7 @@
 
 In order to use the Cronofy API you will need to [create a developer account](https://app.cronofy.com/sign_up/new).
 
-From there you can [create personal access tokens](https://app.cronofy.com/oauth/applications/5447ae289bd94726da00000f/tokens)
+From there you can [Use your Calendar Sandbox](https://app.cronofy.com/oauth/sandbox)
```

Comment:
Are we intending to capitalize this?

So it reads "From there you can Use your Calendar Sandbox"?